app/services/ReturnsService.scala

```diff
@@ -64,13 +64,13 @@ class ReturnsService @Inject()(val vatRegConnector: VatRegistrationConnector,
   }
 
   def handleView(returns: Returns): Completion[Returns] = returns match {
-    case Returns(_, _, _, _, _, _, _, _, _, Some(NIPCompliance(goodsToEU, None))) =>
```

Comment:
I'd change the last None here to _ as this case is only concerned with NIP stuff.